### PR TITLE
feat: upgrade to Gosling.js v0.9.15

### PR DESCRIPTION
**Pull request by manzt**
Updates heatmap examples. Looking forward to some defaults for `responsiveSize` somehow.

### CODE REVIEW

File: gosling/examples/heatmap.py [hunk 1]
```diff
@@ -9,10 +9,12 @@
...
 # data = gos.matrix('/path/to/dataset.cool') # local dataset
     
-track = gos.Track(data).mark_rect().encode(
-  x="position1:G",
-  y="position2:G",
-  color=gos.Color("value:Q", range="warm"),
+track = gos.Track(data).mark_bar().encode(
+  x="xs:G",
+  xe="xe:G",
+  y="ys:G",
```
Comment (sehilyi) [inline gosling/examples/heatmap.py:15]:
```suggestion
  x=gos.X("xs:G", axis="bottom"),
  xe="xe:G",
  y=gos.Y("ys:G", axis="left"),
```

Comment (sehilyi) [reply gosling/examples/heatmap.py:15]: We can set `axis` to display the y-axis. 

I think Gosling.js should display the y-axis by default in the future.

**Review by sehilyi — APPROVED** [review]
Thank you for upgrading Gosling.js!

It looks like `responsiveSize` is not working on Jupyter Notebooks/Labs, and I will need to address several issues. For example, those notebooks do not seem to set a fixed width to a cell container (similar issues on Altair, https://github.com/altair-viz/altair/issues/2057). Also, I would need to double-check if `.embed()` function properly turns on our responsive options. If I set a fixed width to several containers of Gosling visualization, the responsive sizing works correctly.

It looks like the matrix example, which is another part of v0.9.15, is working well. I think we can merge this and support `responsiveSize` in the future.

**Comment by sehilyi** [comment]
@manzt Also, could you elaborate on what you mean by "supporting some defaults for `responsiveSize`"?

**Comment by manzt** [comment]
> It looks like the matrix example, which is another part of v0.9.15, is working well. I think we can merge this and support responsiveSize in the future.

Agreed. Something to work on for the future, better to have `gos` in sync with Gosling.js

**Comment by manzt** [comment]
> @manzt Also, could you elaborate on what you mean by "supporting some defaults for responsiveSize"?

Sure! We've arbitrarily defined fixed defaults for when a width/height aren't defined for a track. I was thinking we could automatically pick a responsive width in the special case where width/height aren't explicitly provided. However, after looking into this a little yesterday, it gets complicated quickly due to how tracks may be composed.

**Comment by sehilyi** [comment]
Thanks for the clarification! 

I think we can at least make the `width` and `height` optional so that users do not have to specify them when users use `responsiveSize`. But, yes, it is somewhat challenging to provide non-constant optimal `width` and `height` to individual tracks when the track/view structure becomes complicated.

Also, I was thinking that the appearance of some visual elements can be adaptive by default depending on the assigned size (e.g., hiding a color legend when the track height is too narrow). Another thing that we can figure out to make the `responsiveSize` more useful.

**Comment by manzt** [comment]
Great ideas. I'll open an issue and merge this.

**Pull request merged**